src/lib.rs

```diff
@@ -111,7 +111,8 @@ impl<L: LogLevel> Verbosity<L> {
         self.log_level().is_none()
     }
 
-    fn verbosity(&self) -> i8 {
+    // The raw i8 value of the combined verbosity level
+    pub fn verbosity(&self) -> i8 {
```

Comment:
Discussion in #53 needs resolving before we can move forward with this

Reply:
(while redundant with @matthiasbeyer comment, this will help mark it as not ready for merging so we don't lose track of this)